tor/validation/formatting_validation.py

```diff
@@ -42,6 +42,12 @@
 # ```
 FENCED_CODE_BLOCK_PATTERN = re.compile("```.*```", re.DOTALL)
 
+# Regex to recognize unescaped hashtags which may render as headers.
+# Example:
+#
+# #Hashtag
+UNESCAPED_HEADING_PATTERN = re.compile(r"(\n[ ]*\n[ ]{,3}|^)#[^ ].*")

```

Comment:
I believe the `.*` at the end can be removed without changing the semantics

Reply:
Yeah, you're right